images/dotnet/Alpine/entrypoint.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
+dotnet ${APP_NAME}.dll
```

Comment:
May not need entrypoint.sh if its just one line like this. Could just change ENTRYPOINT to this: 

ENTRYPOINT dotnet ${APP_NAME}.dll

I think

Reply:
updated as per your feedback